package.json

```diff
@@ -5,30 +5,13 @@
   "main": "index.js",
   "scripts": {
     "test:open": "npm run lint && cypress open",
-    "pretest": "rm -rf cypress/reports/*",
     "lint": "eslint ./cypress/e2e/**/*.ts",
     "lint:fix": "npm run lint -- --fix",
-    "test": "cypress run",
-    "combine:reports": "mochawesome-merge cypress/reports/mocha/*.json > cypress/reports/report.json",
-    "generate:reports": "marge cypress/reports/report.json -f report -o cypress/reports"
+    "test": "cypress run"
+
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/mafeZapata/cypress-training.git"
-  },
-  "keywords": [
-    "ui-testing",
-    "dojo",
-    "practice",
-    "cypress"
-  ],
-  "author": "Maria Fernanda Zapata mafez0427@gmail.com mafeZapata",
-  "license": "MIT",
-  "bugs": {
-    "url": "https://github.com/mafeZapata/cypress-training/issues"
-  },
-  "homepage": "https://github.com/mafeZapata/cypress-training#readme",
-  "devDependencies": {
```

Comment:
why you delete this?

Reply:
I have a problem with the CI with the lint but i upload the file and resolve this problem